packages/plugin-select/src/select/select.jsx

```diff
@@ -61,7 +61,8 @@ export default function select(...selectors) {
       }
 
       render() {
-        return <Component {...this.state} {...this.props} />;
+        const { _forwardedRef, ...restProps } = this.props;
+        return <Component {...this.state} ref={_forwardedRef} {...restProps} />;
```

Comment:
The ref prop would be the last one defined properties.

Reply:
https://github.com/seznam/IMA.js-plugins/pull/14/commits/af8a31b2ff0a5bf562f56869766f7013e4ee78dc